ui/src/stores/indexedDB.ts

```diff
@@ -70,13 +71,36 @@ export async function openDatabase(): Promise<IDBDatabase> {
     request.onsuccess = async (event) => {
       const db = (event.target as IDBOpenDBRequest).result;
       await ensureInitialState(db);
+      const general = await loadStateFromIndexedDB("general", db);
+      const shouldClearClipboard = await isClipboardTimeoutExpired(general);
+
+      if (shouldClearClipboard && general?.clipboard) {
+        general.clipboard = undefined;
+        saveStateToIndexedDB({ clipboard: general.clipboard }, "general");
```

Comment:
Please be more conscious of when you might be mutating state outside of locally scoped variables
```suggestion
        saveStateToIndexedDB({ clipboard: undefined }, "general");
```